iron-form.html

```diff
@@ -23,13 +23,15 @@
   Example:
 
     <iron-form>
-      <form method="get" action="/form/handler">
+      <form method="get" action="/form/handler" enctype="application/json">
```

Comment:
Does this need to be here? This example to me implies that you _always_ have to specify it.

Reply:
No it is not necessary, I opened this PR before https://github.com/PolymerElements/iron-form/pull/213 - at that time enctype was necessary because the default behavior was incorrect.